src/main/scala/io/opentargets/openfda/ETL.scala

```diff
@@ -15,18 +15,21 @@ object ETL extends LazyLogging {
         logger.info("run step fda pipeline...")
         val fdaConfig = context.configuration.fda
         logger.info("Aggregating FDA data...")
-        val openFdaDataAggByChembl: DataFrame = OpenFdaEtl.apply
+        val openFdaDataAggByChembl: DataFrame = OpenFdaEtl.apply.cache()
```

Comment:
I've tried the pipeline without persisting the intermediate result for `openFdaDataAggByChembl` and we see the double hump on the metrics graph. Without persisting the result we calculate the result once when mcResults is evaluated, and a second time when we write out the `aggByChembl` results. Therefore I'll change to call from `cache()` to `persist(StorageLevel.MEMORY_AND_DISK_SER)` unless there is a better solution to avoid recomputing the result.